frontend/src/components/camp/CollaboratorListItem.vue

```diff
@@ -107,10 +107,9 @@ export default {
   methods: {
     resendInvitation () {
       this.resendingEmail = true
-      this.api.href(this.api.get(), 'invitation', {
-        action: 'resend',
-        campCollaborationId: this.collaborator.id
-      }).then(postUrl => this.api.post(postUrl, {}))
+      this.api.href(this.api.get(), 'campCollaborations', {
+        id: this.collaborator.id
+      }).then(postUrl => this.api.patch(postUrl + '/resend_invitation', {}))
```

Comment:
Hast du schon eine Ahnung wie aufwändig es ist, die RPCs im API root zu ergänzen, oder soll ich mir das mal anschauen?

Reply:
Darfst du sonst anschauen

Reply:
:+1: Mach ich dann, nachdem usu die UriTemplateFactory bei sich eingebaut und ev. angepasst hat.
Es ist nämlich noch nicht klar, wie die RPC operations genau ausgewiesen werden sollen im API root. Optionen:
```
user: {
  href: '/users{/:id}', templated: true
},
userActivate: {
  href: '/users/{:id}/activate', templated: true
},
  
```
Oder
```
user: {
  href: '/users/{:id}{/:action}', templated: true
},
  
```
Oder
```
user: {
  href: '/users/{:id}{/activate,deactivate,someotheraction}', templated: true
},
  
```
Muss noch herausfindenwas in URI templates überhaupt erlaubt ist.